srcs/execution/builtins/setenv.c

```diff
@@ -12,12 +16,17 @@ BUILTIN_RET		builtin_setenv(BUILTIN_ARGS)
 
 	if (argc == 3)
 	{
-		setenv_as(env, argv[1], argv[2]);
-		return (STATUS_SUCCESS);
+		if (ft_strchr(argv[2], '='))
```

Comment:
The  unsetenv() function shall ........  The named argument shall not contain an '=' character
source http://www.unix.com/man-page/posix/3p/unsetenv/